streams/src/test/java/org/apache/kafka/streams/processor/internals/tasks/DefaultTaskManagerTest.java

```diff
@@ -79,7 +79,7 @@ public void shouldShutdownTaskExecutors() {
         taskManager.shutdown(duration);
 
         verify(taskExecutor).requestShutdown();
-        verify(taskExecutor).awaitShutdown(duration);
+        verify(taskExecutor).awaitShutdown(Duration.ofSeconds(10));
```

Comment:
This change may need to be reverted. Both JUnit tests Java 11 & 17 failed on this case.